students/marsha_w/lesson02/grid_printer_exercise.py

```diff
@@ -0,0 +1,21 @@
+'''
+Title:Grid Printer Exercise
+Dev:Marsha Wheeler
+Date:01-23-18
+'''
+
+#Write a function that draws a Grid and uses two parameters in the function
+def drawGrid(nRows, nCells):
+    plus = '+'
+    minus = '-'
+    bar = '|'
+    line1 = plus + minus * nCells + plus + minus * nCells + plus # concatenate strings
+    line2= bar +' ' * nCells + bar +' ' * nCells + bar # concatenate strings
+    for i in range(0,nRows):
```

Comment:
no need for the `0` in your range.  it will start at 0 index by default `for i in range(nRows):`